addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -107,6 +147,21 @@ const KnowledgeArticleFormRenderer = FormRenderer.extend(KnowledgeTreePanelMixin
             }
         });
     },
+    /**
+     * Update tbe btn-chatter appearance and show the chatter section in the
+     * view. Open the chatter.
+     *
+     * @private
+     * @param {OdooEvent} ev
+     */
+    _onOpenChatter: async function (ev) {
```

Comment:
When I open and close the chatter, I see in the network panel that the script does rpc calls. Is it possible to only fetch the data on the first click and then simply hide/show the div on the following clicks ? The idea would be to avoid re-fetching the data from the server each time the user clicks on the icon.

Here, I'm just throwing some ideas: Maybe, you do want to re-fetch the data from the server to show the latest info to the user as there is no auto-resync mechanism with the chatter. In that case, your suggestion seems like a viable and simple solution :smile: So do not implement the idea without discussing with others.

Reply:
Agree that a better async chatter would probably be interesting (aka know if it is outdated or not). But this belongs more to a specific task than a fix :) .